Makefile

```diff
@@ -0,0 +1,32 @@
+JSONNET_FMT := jsonnet fmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
```

Comment:
If these arguments fallback to the default, I don't think we need to set them explicitly. :slightly_smiling_face: